.github/workflows/build-test.yaml

```diff
@@ -464,12 +467,14 @@ jobs:
         with:
           test-focus: 'Smoke Test'
           kots-namespace: 'smoke-test'
+          platform: '${{ matrix.platform }}'
           k8s-version: '${{ matrix.k8s_version }}'
           k8s-arguments: ${{ startsWith(matrix.k8s_version, 'v1.22') && '--egress-selector-mode disabled' || '' }}
           testim-access-token: '${{ secrets.TESTIM_ACCESS_TOKEN }}'
           testim-branch: ${{ github.head_ref == 'main' && 'master' || github.head_ref }}
           aws-access-key-id: '${{ secrets.E2E_SUPPORT_BUNDLE_AWS_ACCESS_KEY_ID }}'
           aws-secret-access-key: '${{ secrets.E2E_SUPPORT_BUNDLE_AWS_SECRET_ACCESS_KEY }}'
+          gke-service-key: ${{ secrets.GOOGLE_CLOUD_CREDENTIALS }}
```

Comment:
why is this needed with k3s? (same to most other tests that are only run on k3s)

Reply:
Craig pointed this out too, I originally added it so that it wouldn't be needed to add gke tests, but it does feel weird passing around an unused secret, so the excess ones are removed now